pkg/trace/config.go

```diff
@@ -21,13 +21,15 @@ func (g *GlobalTags) MarshalLog(addField func(key string, v interface{})) {
 }
 
 type Otel struct {
-	Enabled       bool       `yaml:"Enabled"`
-	Endpoint      string     `yaml:"Endpoint"`
-	Dataset       string     `yaml:"Dataset"`
-	SamplePercent float64    `yaml:"SamplePercent"`
-	Debug         bool       `yaml:"Debug"`
-	Stdout        bool       `yaml:"Stdout"`
-	APIKey        cfg.Secret `yaml:"APIKey"`
+	Enabled            bool       `yaml:"Enabled"`
```

Comment:
nit: Can we get a comment on each of these, lintroller plat will want it in the future